app/views/wpcc/your_results/index.html.erb

```diff
@@ -1,15 +1,35 @@
 <%= render 'wpcc/shared/your_details' %>
 <%= render 'wpcc/shared/your_contributions', message_presenter: message_presenter %>
-
 <section class="section section--results">
   <h2 class="section__heading results__heading">3. <%= t('wpcc.results.title') %></h2>
 
   <div class="results__content" data-dough-component="UpdateResults">
-    <p>
-      <%= t('wpcc.results.description_html') %>
-      <%= your_results_presenter.earnings_description.html_safe %>
-      <%= period_filter_presenter.contribution_percents_explanation %>
-    </p>
+    <div data-dough-component="PopupTip">
+      <p>
+        <%= t('wpcc.results.description_html') %>
+        <% if message_presenter.minimum_contribution? %>
+          <% tooltip_html = render('wpcc/tooltips/trigger', locals: { popup_type: 't-qualifying-earnings' }) %>
+          <%= your_results_presenter.earnings_description({ tooltip_html: raw(tooltip_html) }) %>
+        <% else %>
+          <%= your_results_presenter.earnings_description.html_safe %>
+        <% end %>
+        <%= period_filter_presenter.contribution_percents_explanation %>
+      </p>
+
+      <div data-dough-popup-container class="popup-tip__container results__helper">
+        <p data-dough-popup-content class="popup-tip__content">
+          <span class="popup-tip__title--no-js">
+            <%= t('wpcc.tooltip_qualifying_earnings.title') %>:
+          </span>
+          <%= t(
+            'wpcc.tooltip_qualifying_earnings.text',
+            formatted_upper_earnings: message_presenter.formatted_upper_earnings,
+            formatted_lower_earnings: message_presenter.formatted_lower_earnings
+            ) %>
+        </p>
+        <%= render 'wpcc/tooltips/close' %>
+      </div>
+    </div>
```

Comment:
see comment above re partialising the tooltop

Reply:
@margOnline Yes, that would make some sense TBH. I'll look into it.

Reply:
@margOnline I've added a partial for the tooltip content. Only issue with this is that the values for formatted_upper_earnings and formatted_lower_earnings need to be passed through as well as the class name because they are derived differently on each view. Can you have a look and see if this is OK or if there is a better way to approach it? Cheers.

Reply:
@davidtrussler one small thing, in ruby the default is snake_case, so the partial should be named `_content_qualifying_earnings` or maybe `_qualifying_earnings_message`; otherwise looks good